src/sentry/tasks/derive_code_mappings.py

```diff
@@ -166,9 +119,10 @@ def get_installation(organization: Organization) -> Tuple[Integration, Organizat
 
 def set_project_codemappings(
     code_mappings: List[CodeMapping],
-    organization: Organization,
     organization_integration: OrganizationIntegration,
+    organization_id: int,
```

Comment:
Passing just the `organization_integration` might be sufficient since you can get the associated Organization object, thus, having access to the id:
https://github.com/getsentry/sentry/blob/43b71c18886d2b477e7fedc2eb827f54006b482c/src/sentry/models/integrations/organization_integration.py#L17